tests/pytest/enrollment_littlepay/test_views.py

```diff
@@ -12,6 +34,170 @@ def handler(request, status, exception):
     return handler
 
 
+@pytest.mark.django_db
+def test_token_ineligible(client):
+    path = reverse(routes.ENROLLMENT_LITTLEPAY_TOKEN)
+
+    response = client.get(path)
+
+    assert response.status_code == 200
+    assert response.template_name == TEMPLATE_USER_ERROR
+
+
+@pytest.mark.django_db
+@pytest.mark.usefixtures("mocked_session_agency", "mocked_session_eligible")
+def test_token_refresh(mocker, client):
+    mocker.patch("benefits.enrollment_littlepay.session.Session.access_token_valid", return_value=False)
+
+    mock_token = {}
+    mock_token["access_token"] = "access_token"
+    mock_token["expires_at"] = time.time() + 10000
+
+    mocker.patch(
+        "benefits.enrollment_littlepay.views.request_card_tokenization_access",
+        return_value=CardTokenizationAccessResponse(
+            Status.SUCCESS,
+            access_token=mock_token["access_token"],
+            expires_at=mock_token["expires_at"],
+        ),
+    )
+
+    path = reverse(routes.ENROLLMENT_LITTLEPAY_TOKEN)
+    response = client.get(path)
+
+    assert response.status_code == 200
+    data = response.json()
+    assert "token" in data
+    assert data["token"] == mock_token["access_token"]
+
+
+@pytest.mark.django_db
+@pytest.mark.usefixtures("mocked_session_agency", "mocked_session_eligible")
+def test_token_valid(mocker, client):
+    mocker.patch.object(Session, "access_token", "enrollment_token")
```

Comment:
This was the adjustment:

Previously this test used the `@patch()` decorator from `unittest.mock` to override the `Session.access_token` attribute with a `PropertyMock` (also from `unittest.mock`).

The small change was to use `mocker.patch.object` on `Session` to achieve the same result.

Reply:
Makes sense! Thanks for the adjustment